dashboard/dashboard-ops-deploy.md

```diff
@@ -14,7 +14,7 @@ The TiDB Dashboard UI is built into the PD component for v4.0 or higher versions
 
 See the following documents to learn how to deploy a standard TiDB cluster:
 
-+ [Quick Start Guide for the TiDB Database Platform](/quick-start-with-tidb.md)
++ [Quick Start with TiDB Self-Managed Using TiUP Playground](/quick-start-with-tidb.md)
```

Comment:
Using -> using

Reply:
We use the Chicago Style for titles, so let's keep it as "Using".